debian/stocks-client.postinst

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Just to be sure: This script must be executed after every update, not only installation. Sometimes the client DB cannot be migrated in which case it will be deleted and copied over by the CLI client automatically. So the new DB should then contain the latest schema.

Reply:
Yes, this is the case. The actual schema is pretty complicated, see here: https://wiki.debian.org/MaintainerScripts.